charts/handbrake/1.0.0/test_values.yaml

```diff
@@ -0,0 +1,127 @@
+image:
+  repository: jlesage/handbrake
+  tag: v1.23.1
+  pullPolicy: IfNotPresent
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 5800            #Required for web GUI
+
+#All values here are set as the docker defaults.
+env:
+# Permissions Settings
+  USER_ID: "{{ .Values.PUID }}"
+  GROUP_ID: "{{ .Values.PGID }}"
```

Comment:
no, thats not what I asked, try again.
DONT use env for this. See before, its an env template, not an env.

Reply:
Should be ok now, sorry

Reply:
No problem, just here to help you get this to pass :)